collect_app/src/test/java/org/odk/collect/android/projects/AddProjectDialogTest.kt

```diff
@@ -32,8 +39,22 @@ class AddProjectDialogTest {
         val scenario = RobolectricHelpers.launchDialogFragment(AddProjectDialog::class.java)
         scenario.onFragment { f: AddProjectDialog ->
             val dialog = f.dialog
-            Espresso.onView(ViewMatchers.isRoot()).perform(ViewActions.pressBack())
+            onView(isRoot()).perform(ViewActions.pressBack())
             assertThat(dialog!!.isShowing, `is`(false))
         }
     }
+
+    @Test
+    fun oneCharacterOrEmojiOnly_shouldBeAcceptedAsProjectIcon() {
```

Comment:
I like this. Definitely good to check the emoji case here, so we don't accidentally break that! It'd also be good to add a test here that covers adding more than one character (it looks there is already implementation for that).